### ISSUE TITLE
how to customized by passing proper value to ReactInstanceManager#setUseDeveloperSupport ?

### ISSUE THREAD

**Issue by suhaotian** [facebook/react-native#6932]
how to customized by passing proper value to ReactInstanceManager#setUseDeveloperSupport ?

please tell me

**Comment by dsibiski** [comment]
Hi @suhaotian, can you explain a little more what it is you're asking for?

**Comment by suhaotian** [comment]
Hi @dschleimer , when I  **./gradle assembleRelease**, the debug menu never show on android. The Doc [here](http://facebook.github.io/react-native/releases/0.23/docs/debugging.html#debugging-react-native-apps) say: 
** For Android, by default, developer menu will be disabled in release builds done by gradle (e.g with gradle assembleRelease task). Although this behavior can be customized by passing proper value to ReactInstanceManager#setUseDeveloperSupport. **

But I don't know how to ** customized by passing proper value to ReactInstanceManager#setUseDeveloperSupport. ** please tell me how to do it . thank you ,very much

**Comment by dsibiski** [comment]
@suhaotian The `assembleRelease` command is only for "Release mode" when you want to submit the app to Google Play. If you want the debug menu to show you simply run `react-native run-android`.

**Comment by suhaotian** [comment]
@dsibiski  Got it :smile:  thanks!

**Comment by edudel** [comment]
I can't see the dev menu yet (android). IOS works
RN 0.48.3. (newest)
how can I get the real time changes after using the bundle ?
I got this after upgrading react
Unable to load script from assets 'index.android.bundle'. Make sure your bundle is packaged correctly or you're running a packager server.
so I had to do execute 
react-native bundle --platform android --dev true --entry-file index.android.js .........
etc
now the dev menu is gone in android.. even when I run react-native run-android, react-native run-android --variant=debug etc etc 

Any ideas ?
